utilities/pipelines/dependencies/Microsoft.Storage/storageAccounts/parameters/synapse01.parameters.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "value": "adpsxxazsaweux004"
```

Let's rename it with something mapping it to the service for which is needed plus use the token e.g.
```suggestion
      "value": "adp<<namePrefix>>azsasynapse001"
```

---

And they are leveraged as DataLake2 Storage Accounts. For example in file `encryptionwsai.parameters.json` & `managedvnet.parameters.json` respectively

---

So if I got it right we are not deploying dedicated dependencies anymore but using the existing shared storage dependencies 001 and 002?

---

Now I'm confused. The `[synapse01.parameters.json](https://github.com/Azure/ResourceModules/pull/928/files/ed5f0b134d6ce227b53abe6b817e09487c136d4c#diff-d23b02be553d3e71b3607d4e92d3b27153913c18835be3d96907ebb292f14d6e)` file deploys the storage account `adp<<namePrefix>>azsasynapse001` which is used in one of the parameter files. The same for the other.

---

The naming got screwed up. Updated the dependencies & test files

---

Makes sense now! Thanks. Can we add the new dependencies to the Wiki? 👼